packages/publish_npm.sh

```diff
@@ -20,11 +20,31 @@ check_prerequisites ()
         echo "Error: You have local changes, which is not allowed.  Use 'git status' to view."
         return 1
     fi
+
+    # Checks for publishing to Github release
+    regex='"version": "([0-9.]+)",'
+    package=$(cat ./vertica-nodejs/package.json)
+    [[ $package =~ $regex ]]
+    version=${BASH_REMATCH[1]}
+    if [ -z "$version" ]
+    then
+        echo "Error: Cannot detect version to be published."
+        return 1
+    else
+        echo "[Success] Detected version to be published: $version"
+    fi
+
+    if [ -z "$GITHUB_TOKEN" ]
```

Comment:
I hadn't thought about using an environment variable locally. This will save me some time